### DISCUSSION TITLE
Milvus Standalone export to another system

### DISCUSSION THREAD

**Discussion by AmanRavan** [milvus-io/milvus#31773] [Q&A and General discussion] [answered]
I am using Milvus Standalone, installed using binary in windows. I have added a collections in Milvus 
1 - How to share with milvus database other friend on different PC.
2 - Other friend is having Mac or running in via docker compose will that exported Milvus database will work for him.

**Comment by yhmo** [comment, marked as answer]
There is an official tool can backup/restore collections across milvus instances: https://github.com/zilliztech/milvus-backup

**Reply by AmanRavan** [reply]
Do we have any documentation or getting started?

**Reply by yhmo** [reply]
Currently, the doc is not available, we are preparing to write a detailed document for it, the work is not finished, yet.

You can read more background knowledge about milvus-backup tool from this discussion: https://github.com/milvus-io/milvus/discussions/29630
